dynawo/sources/Models/Modelica/Dynawo/Examples/Nordic/TestCases/TestCase.mo

```diff
@@ -0,0 +1,142 @@
+within Dynawo.Examples.Nordic.TestCases;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite
+* of simulation tools for power systems.
+*/
+
+model TestCase "Nordic test system case with variable reference frequency"
+  import Modelica.ComplexMath;
+  import Dynawo;
+  import Dynawo.Electrical.SystemBase;
+  import Dynawo.Types;
+
+  extends Dynawo.Examples.Nordic.Grid.FullDynamicModel;
+  extends Icons.Example;
+
+  Types.AngularVelocityPu omegaCOI(start = 1);
```

Comment:
Please use SystemBase for start

Reply:
Please add a comment to say this is the center of inertia, calculated as a weighted average of all the generators frequencies